<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReaderWindow</class>
 <widget class="QWidget" name="ReaderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="borrowTab">
    <attribute name="title">
     <string>Borrow</string>
    </attribute>
    <widget class="QTableView" name="bookTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>551</width>
       <height>321</height>
      </rect>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="borrowBookSearchEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="borrowBookButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>370</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Borrow</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>370</y>
       <width>194</width>
       <height>26</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy/M/d hh:mm</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ddlLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>373</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Borrow Until:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="returnTab">
    <attribute name="title">
     <string>Return</string>
    </attribute>
    <widget class="QLineEdit" name="returnBookSearchEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="borrowBookTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>551</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="returnBookButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>370</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QPushButton" name="renewBookButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>370</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Renew</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="logTab">
    <attribute name="title">
     <string>Log</string>
    </attribute>
    <widget class="QTableView" name="logTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>551</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="logSearchEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ticketTab">
    <attribute name="title">
     <string>Ticket</string>
    </attribute>
    <widget class="QLineEdit" name="ticketSearchEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="ticketTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>551</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="payButtom">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>370</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Pay</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="nameTitleLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="majorTitleLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Major:</string>
   </property>
  </widget>
  <widget class="QLabel" name="majorLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>300</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="idTitleLabel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>       ID:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
